lab3/artist.php

```diff
@@ -0,0 +1,328 @@
+<?php 
+
+  session_start();
+
+  if(!$_SESSION["loggedin"]) {
+    header("Location: login.php");
+    die();
+  }

```

Comment:
Dit stukje zou je nog in een aparte include kunnen stoppen (en binnenkort in een aparte klasse)